tests/api.c

```diff
@@ -1617,7 +1617,63 @@ static void test_wolfSSL_CTX_SetTmpDH_buffer(void)
     wolfSSL_CTX_free(ctx);
 #endif
 }
+static void test_wolfSSL_DH_get0_pqg(void)
+{
+#if defined(OPENSSL_EXTRA) && !defined(NO_DH)
+    DH *dh = NULL;
+    BIGNUM* p;
+    BIGNUM* q;
+    BIGNUM* g;
+
+#if defined(OPENSSL_ALL)
+#if !defined(HAVE_FIPS) || (defined(HAVE_FIPS_VERSION) && (HAVE_FIPS_VERSION>2))
```

Comment:
Minor style adjustment, please add space around ">" in "(HAVE_FIPS_VERSION > 2)"

Reply:
Adjusted.